src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementProductTable.tsx

```diff
@@ -162,7 +166,10 @@ const ReimbursementProductTable: React.FC<ReimbursementProductTableProps> = ({
                     sx={{
                       margin: '4px',
                       '&:focus': {
-                        backgroundColor: '#fdf8f8'
+                        backgroundColor: hoverColor
```

Comment:
I think we can remove `&focus` prop from `sx` prop to remove the different color background on 'add product' button. It got removed in my local app after I removed `&focus` prop.

Reply:
Removing `&:focus` from the sx prop makes it so there is no indication when we tab through to the add product button. I think that would go against the acceptance criteria for this ticket?

Reply:
I see. According to the ticket description, your point sounds right. However, I don't get the point of us having a highlight show up on the button after we click on `ADD PRODUCT` and `delete icon`.

@RChandler234 Let us know if this is what you expected for this ticket.

Reply:
you can make the onclick method for add product and it'll fix both problems:
```
(e) => {
                      appendProduct({
                        wbsNum: validateWBS(key),
                        name: '',
                        cost: 0
                      });
                      e.currentTarget.blur();
                    }
```